packages/bsky/src/services/indexing/processor.ts

```diff
@@ -57,6 +56,17 @@ export class RecordProcessor<T, S> {
     if (!this.matchesSchema(obj)) {
       throw new Error(`Record does not match schema: ${this.params.lexId}`)
     }
+    await this.db
+      .insertInto('record')
+      .values({
+        uri: uri.toString(),
+        cid: cid.toString(),
+        did: uri.host,
+        json: JSON.stringify(obj),
```

Comment:
need to make sure we use the lex serializer on these 

for cids/bytes/blob refs

Reply:
maybe throw a TODO on or something cause this could end up causing unexpected bugs

Reply:
goes every anywhere that records get serialized to/from json